spec/integration/preview_spec.rb

```diff
@@ -579,4 +579,25 @@
       expect(err).to match_regex(/trusted option is deprecated/)
     end
   end
+
+  context 'when using --skip-inactive-nodes' do
+    env_path = File.join(testdir_simple, 'environments')
+    # deactivate node
+    test_node = "pl-inactive-node-#{rand(999999).to_i}"
+    add_node_to_puppetdb(test_node, 'production')
+    on master, puppet('node', 'deactivate', test_node)
+
+    it 'should skip inactive nodes by default' do
+      result = on(master, puppet("preview --skip-inactive-nodes --preview-environment test --environmentpath #{env_path} --view baseline #{test_node}"),
+        :acceptable_exit_codes => [1])
+      assert_match(/none of the given node\(s\) are active/,result.stderr,'preview output from failed preview did not match expected')
```

Comment:
do we need to test the case where one node is active and another is not, and we get some results for the active node, but not the inactive node?